### PR TITLE
Add action for pushing to quay.io

### PR DESCRIPTION
**Pull request by johnmcollier**
Adds an action to push the application-service operator's image to quay.io/redhat-appstudio/application-service:<sha> (and latest)

### CODE REVIEW

**Comment by johnmcollier** [comment]
Hey @maysunfaisal can you take a look when you get the chance? Thanks!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+
+  build-imgs:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout devworkspace-operator source code
```
Comment (maysunfaisal) [inline .github/workflows/build.yml:27]: typo

Comment (johnmcollier) [reply .github/workflows/build.yml:27]: Copied from [redhat-appstudio/service-provider-integration-operator](https://github.com/redhat-appstudio/service-provider-integration-operator) 😆

Comment (johnmcollier) [reply .github/workflows/build.yml]: Fixed.

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**